raidprotect/src/event/message/handle.rs

```diff
@@ -14,5 +21,41 @@ pub async fn handle_message(message: Message, state: Arc<ClusterState>) {
     let parsed = parse_message(&message);
     state.redis().set(&parsed).await.ok();
 
+    if is_an_old_command(&message) {
+        if let Err(error) = warn_about_old_command(&message, state.http()).await {
+            error!(error = ?error, "failed to warn user about question mark commands deprecation");
+        }
+    }
+
     info!("received message: {}", message.content) // Debug util real implementation
 }
+
+async fn warn_about_old_command(message: &Message, http: &Client) -> Result<()> {
+    let lang = Lang::fallback();
+    let embeds = [EmbedBuilder::new()
+        .title(lang.warning_deprecated_command_style_title())
+        .description(lang.warning_deprecated_command_style_description())
+        .color(COLOR_TRANSPARENT)
+        .build()];
+    http.create_message(message.channel_id)
+        .embeds(&embeds)?
```

Comment:
Style nit: initialize the array where, not in the `embed` variable.

```suggestion
        .embeds(&[embed])?
```